src/Components/ErrorPage.tsx

```diff
@@ -0,0 +1,101 @@
+import { color, Serif } from "@artsy/palette"
+import { garamond } from "Assets/Fonts"
+import React from "react"
+import styled from "styled-components"
+import { Button } from "Styleguide/Elements/Button"
+import { Spacer } from "Styleguide/Elements/Spacer"
+
+interface ErrorPageProps {
+  code: number
+  message?: string
+  detail?: string
+}
+
+interface ErrorCodeBackgroundProps extends React.HTMLProps<HTMLDivElement> {
+  code: Number
+}
+
+export class ErrorPage extends React.Component<ErrorPageProps, null> {
+  render() {
+    const { code, message, detail } = this.props
+    const defaultMessage =
+      code === 404
+        ? "Sorry, the page you were looking for doesn’t exist at this URL."
+        : "Internal Error"
+
+    const detailMessage = message ? `Error Message: ${message}` : detail
+
+    return (
+      <>
+        <StyledErrorCodeBackground code={code}>
```

Comment:
I think these components can be simplfied to `ErrorCodeBackground`, `ErrorDefaultMessage`, and so on. The idiom for using a `Style` prefix is when needing to wrap (override) an existing component; e.g., 

```js
import styled from 'styled-components'
import { SomeComponent } from 'path/to/SomeComponent'

const StyledSomeComponent = styled(SomeComponent)`
  ...
`

<StyledSomeComponent />
```

Reply:
Oh! I can get behind that (and wasn't sure about the convention).